csp-mixin/rows.libsonnet

```diff
@@ -216,5 +216,51 @@ local g = import './g.libsonnet';
         + g.panel.gauge.gridPos.withW(12)
         + g.panel.gauge.gridPos.withH(8),
       ],
+
+      // Azure Virtual Network
+      vn_overview: [
+        g.panel.row.new("Overview"),
+        this.grafana.panels.vn_under_ddos
+        + g.panel.timeSeries.gridPos.withW(6)
+        + g.panel.timeSeries.gridPos.withH(6),
+
+        this.grafana.panels.vn_pingmesh_avg
+        + g.panel.timeSeries.gridPos.withW(6)
+        + g.panel.timeSeries.gridPos.withH(6),
+
+        this.grafana.panels.vn_packet_trigger
+        + g.panel.timeSeries.gridPos.withW(12)
+        + g.panel.timeSeries.gridPos.withH(6),
+      ],
```

Comment:
btw, you can now define all panels INSIDE row, like so:
https://github.com/grafana/jsonnet-libs/blob/master/kafka-observ-lib/rows.libsonnet#L6-L24

this was, you can decide if you want specific row to be collapsed or not collapsed and render less important rows as collapsed by by using g.util.panel.resolveCollapsedFlagOnRows() func:
https://github.com/grafana/jsonnet-libs/blob/master/kafka-observ-lib/dashboards.libsonnet#L17-L26

Reply:
Ahh nice! I don't really need any of these rows collapsed tho. Is there any other obvious reason I should consider the new format?